CONTRIBUTING.md

```diff
@@ -60,6 +60,33 @@ functional changes into one commit. When writing commit messages, adhere to
 [widely used
 conventions](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
 
+When the commit fixes a bug, put a message in the body of the commit message
+pointing to the number of the issue (e.g. "Fixes #123").
```

Comment:
I normally prefer to put it in the Pull Request so the bug isn't spammed by regular updates of the branch. I think then it would also end up in the Merge Commit description.